webcrypto/subtle_crypto.go

```diff
@@ -633,14 +632,29 @@ func (sc *SubtleCrypto) ImportKey(
 	rt := sc.vu.Runtime()
 	promise, resolve, reject := promises.New(sc.vu)
 
+	var keyBytes []byte
+
 	// 2.
-	ab, err := exportArrayBuffer(rt, keyData)
-	if err != nil {
-		reject(err)
-		return promise
+	switch format {
+	case RawKeyFormat:
+		ab, err := exportArrayBuffer(rt, keyData)
+		if err != nil {
+			reject(err)
+			return promise
+		}
+
+		keyBytes = make([]byte, len(ab))
+		copy(keyBytes, ab)
+	case JwkKeyFormat:
+		var err error
+		keyBytes, err = json.Marshal(keyData.Export())
+		if err != nil {
+			reject(NewError(ImplementationError, "wrong keyData format for JWK format: "+err.Error()))
+			return promise
+		}
+	default:
+		reject(NewError(ImplementationError, "unsupported format "+format))
```

Comment:
```suggestion
		reject(NewError(ImplementationError, "unsupported format "+format))
		return promise
```

Shouldn't we return the promise here as well, as we do for the rest of the error cases? 🤔

Reply:
Yes, we should! Great catch!